test/e2e/network/kube_proxy.go

```diff
@@ -171,16 +171,16 @@ var _ = SIGDescribe("Network", func() {
 		// If test flakes occur here, then this check should be performed
 		// in a loop as there may be a race with the client connecting.
 		framework.IssueSSHCommandWithResult(
-			fmt.Sprintf("sudo cat /proc/net/ip_conntrack | grep 'dport=%v'",
+			fmt.Sprintf("sudo cat /proc/net/nf_conntrack | grep 'dport=%v'",
 				testDaemonTcpPort),
 			framework.TestContext.Provider,
 			clientNodeInfo.node)
 
 		// Timeout in seconds is available as the third column from
```

Comment:
The columns have changed in `nf_conntrack`. It should be the 5th one.

Reply:
Ah, it would be the third field for the "conntrack" untility's output.

Perhaps that is an argument towards using it to abstract eventual kernel changes?

Let me know what you think is appropriate here.

Reply:
`conntrack` is not installed on everywhere - `cos-63` does not have it. Looks like checking `nf_conntrack` is safer.

Reply:
Ack. I'll update the PR in that sense.